docker/mariadb/my.cnf

```diff
@@ -1,2 +1,3 @@
 [mysqld]
-bind-address = 0.0.0.0
\ No newline at end of file
+bind-address = 0.0.0.0
+innodb_temp_data_file_path = ../../../tmp/ibtmp1:12M:autoextend
```

Comment:
This is done because I'm running my cluster on a series on Raspberry Pis and using Longhorn NFS which is too slow for the temp directory. This is documented in mariadb.